### PR TITLE
Forgot password

### PR DESCRIPTION
**Pull request by brian-david**

### CODE REVIEW

**Comment by christoff-linde** [comment]
image

Just some small issues:
- The alert shows up as soon as you open the forgot-password screen (without any user input)
- There are some styling issues

File: src/components/Auth/ForgotPassword.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+  const signUpForm = (
+    <div className="card">
+      <div className="card-content">
+        <div className="content">
+          <div className="form">
+            <Alert
+              message={"We're sent you a confirmation code, check your email"}
+              notificationType={NotificationType.isSuccess}
+            />
```
Comment (christoff-linde) [inline src/components/Auth/ForgotPassword.tsx:53]: I suspect this duplicate `Alert` component is causing the issue

File: src/components/Auth/SignIn.tsx [hunk 2]
```diff
@@ -44,6 +45,23 @@ const SignIn: React.FC = () => {
...
+    } else {
+      forgotPassword(formValues.username)
+        .then(
+          // eslint-disable-next-line no-console
+          data => console.log(data)
+        )
+        .catch(err => {
+          // eslint-disable-next-line no-console
+          console.log(err)
+        })
```
Comment (christoff-linde) [inline src/components/Auth/SignIn.tsx:61]: Why do we need these `console.log` statements?

**Review by christoff-linde — CHANGES_REQUESTED** [review]

File: src/components/Auth/ForgotPassword.tsx (hunk 1 above)
Comment (brian-david) [reply src/components/Auth/ForgotPassword.tsx:53]: So when you press forgot password on the signin screen the system sends the user a verification code via email. That's what the alert is for. They need to know that they need that code to reset the password

File: src/components/Auth/SignIn.tsx (hunk 2 above)
Comment (brian-david) [reply src/components/Auth/SignIn.tsx:61]: mistake

**Comment by ksamson2** [comment]
Handled in a different PR

**Pull request closed without merging**